src/SFA.DAS.ApplyService.Web/Controllers/Roatp/RoatpApplicationPreambleController.cs

```diff
@@ -399,8 +399,25 @@ public async Task<IActionResult> SelectApplicationRoute()
             return View("~/Views/Roatp/SelectApplicationRoute.cshtml", model);
         }
 
+        [HttpPost]
         public async Task<IActionResult> ProcessRoute(SelectApplicationRouteViewModel model)
         {
+            if (!ModelState.IsValid)
+            {
+                model.ApplicationRoutes = await GetApplicationRoutes();
```

Comment:
Why is the standard way of reporting validation errors not being used here?

Reply:
View requires ApplicationRoutes, which is not there in the model until we fill it